apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -117,6 +118,34 @@ txs_gc(Config) ->
 
     ok = aecore_suite_utils:check_for_logs([dev1], Config).
 
+
+micro_block_cycle(Config) ->
+    MBC = aec_governance:micro_block_cycle(),
+    aecore_suite_utils:start_node(dev1, Config),
+    N1 = aecore_suite_utils:node_name(dev1),
+    aecore_suite_utils:connect(N1),
+
+    %% Mine a block to get some funds. Height=1
+    aecore_suite_utils:mine_key_blocks(N1, 1),
+
+    StartRes = rpc:call(N1, aec_conductor, start_mining, [], 5000),
+    ct:log("aec_conductor:start_mining() (~p) -> ~p", [N1, StartRes]),
+
+    [ begin
+        add_spend_tx(N1, 1000, 1,  Nonce, 10000),
+        timer:sleep(MBC div 100)
+      end || Nonce <- lists:seq(1,20) ],
+
+    timer:sleep(2*MBC),
+    StopRes = rpc:call(N1, aec_conductor, stop_mining, [], 5000),
+    ct:log("aec_conductor:stop_mining() (~p) -> ~p", [N1, StopRes]),
+
+    MicroBlocks = aecore_suite_utils:events_since(N1, micro_block_created, ?config(tc_start, Config)),
```

Comment:
Event delivery seems keeping order of publishing, and proxy performs appends to the dict entry for that event (hence keeping order). So seems fine.

Reply:
I have one moment been considering to sort the list but found it unnecessary.